Explorer/Assets/DCL/ExplorePanel/ExplorePanelView.cs

```diff
@@ -50,20 +53,19 @@ private void Awake()
 
         protected override UniTask PlayShowAnimation(CancellationToken ct)
         {
+            CanvasGroup.alpha = 0;
             if (snapshotsPresent) { MuteSoundsSnapshot.TransitionTo(2); }
             UIAudioEventsBus.Instance.SendPlayLoopingAudioEvent(BackgroundMusic);
             UIAudioEventsBus.Instance.SendPlayAudioEvent(OpenMenu);
-            AnimationTransform.anchoredPosition = new Vector2(0, canvas.pixelRect.width);
-            return AnimationTransform.DOAnchorPos(Vector2.zero, 0.5f).SetEase(Ease.OutCubic).ToUniTask(cancellationToken: ct);
+            return CanvasGroup.DOFade(1, 0.2f).SetEase(Ease.Linear).ToUniTask(cancellationToken: ct);

```

Comment:
Maybe expose de 0.2f so it can be tweaked? although if this wont ever be changed it could also be a constant

Reply:
Done